### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
* Added Cameron
* Removed people no longer on team

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -9,11 +9,10 @@ approvers:
 - kevinsalerno
 - kschumy
 - markapruett
-- mdsgabriel
 - mitalipaygude
-- panktishah26
```
Comment (jaxesn) [inline OWNERS:14]: You can leave pankti

**Review by markapruett — APPROVED** [review]

File: OWNERS (hunk 1 above)
Comment (kschumy) [reply OWNERS:14]: # 🤦‍♀️

File: OWNERS [hunk 2]
```diff
@@ -9,11 +9,10 @@ approvers:
...
 - pokearu
```
Comment (kschumy) [inline OWNERS:13]:
```suggestion
- panktishah26
- pokearu
```

**Pull request merged**